src/handlers/tokenLocksHandler.ts

```diff
@@ -10,18 +10,33 @@ import { handleError } from "../response";
 
 const prisma = new PrismaClient();
 
-const tokenLockResponse = (transaction) => ({
+const commonTokenLockResponse = (transaction) => ({
   currencyId: transaction.currencyId,
   hash: transaction.hash,
   amount: transaction.amount,
   source: transaction.source_addr,
   destination: transaction.destination_addr,
   unlockEpoch: transaction.unlock_epoch,
   parentHash: transaction.parent_hash,
+  ordinal: transaction.ordinal,
   timestamp: transaction.created_at,
 });
 
-const tokenLockResponses = (txs) => txs.map(tokenLockResponse);
+const dagTokenLockResponse = (transaction) => ({
+  ...commonTokenLockResponse(transaction),
+  unlockedAtOrdinal: transaction.dag_token_unlock?.global_snapshot.ordinal,
+});
+
+const dagTokenLockResponses = (txs) => txs.map(dagTokenLockResponse);
+
+const metagraphTokenLockResponse = (transaction) => ({
+  ...commonTokenLockResponse(transaction),
+  unlockedAtOrdinal:
```

Comment:
This should default to `null` if not defined so that the field is returned rather than dropped.